Description: Fix upstream installation paths 
Author: Cosimo Alfarano <kalfa@debian.org>
Upstream provides some Makefiles.
Fix upstream patch which are wrong for Debian

Generated with "dpkg-source --commit"
Forwarded: not-needed
Last-Update: Tue Nov 22 13:50:56 GMT 2011

--- gocr-0.49.orig/Makefile
+++ gocr-0.49/Makefile
@@ -10,9 +10,9 @@
 #      make install DESTDIR=$RPM_BUILD_ROOT
 #
 
-prefix = /usr/local
+prefix = /usr
 #bindir = $(prefix)/bin
-bindir = /usr/local${exec_prefix}/bin
+bindir = /usr${exec_prefix}/bin
 
 SHELL = /bin/sh
 # this does not work on SuSE 6.0, why? (autoconf 2.12 do not set /bin/sh ?)
@@ -21,7 +21,7 @@ INSTALL = /usr/bin/install -c
 CC = gcc
 CFLAGS = -g -O2
 # following DEFs are not used, but I play with it for training
-CPPFLAGS = -Wall -g -fexceptions -DDO_DEBUG=1 
+CPPFLAGS = 
 DEFS = -DHAVE_CONFIG_H
 includedir = ${prefix}/include
 #
@@ -100,7 +100,7 @@ help:
 
 
 # you can override this variable by: make install DESTDIR=yourpath
-DESTDIR=/usr/local
+DESTDIR=/usr
 .PHONY : install uninstall test examples
 install:
 	$(MAKE) -C src install
--- gocr-0.49.orig/include/config.h
+++ gocr-0.49/include/config.h
@@ -1,36 +1,37 @@
+/* include/config.h.  Generated by configure.  */
 /* include/config.h.in.  Generated automatically from configure.in by autoheader.  */
 
 /* Define to empty if the keyword does not work.  */
-#undef const
+/* #undef const */
 
 /* Define if the setvbuf function takes the buffering type as its second
    argument and the buffer pointer as the third, as on System V
    before release 3.  */
-#undef SETVBUF_REVERSED
+/* #undef SETVBUF_REVERSED */
 
 /* Define if you have the ANSI C header files.  */
-#undef STDC_HEADERS
+#define STDC_HEADERS 1
 
 /* Define if you have the gettimeofday function.  */
-#undef HAVE_GETTIMEOFDAY
+#define HAVE_GETTIMEOFDAY 1
 
 /* Define if you have the popen function.  */
-#undef HAVE_POPEN
+#define HAVE_POPEN 1
 
 /* Define if you have the wcschr function.  */
-#undef HAVE_WCSCHR
+#define HAVE_WCSCHR 1
 
 /* Define if you have the wcsdup function.  */
-#undef HAVE_WCSDUP
+#define HAVE_WCSDUP 1
 
 /* Define if you have the <pam.h> header file.  */
-#undef HAVE_PAM_H
+#define HAVE_PAM_H 1
 
 /* Define if you have the <pnm.h> header file.  */
-#undef HAVE_PNM_H
+#define HAVE_PNM_H 1
 
 /* Define if you have the <unistd.h> header file.  */
-#undef HAVE_UNISTD_H
+#define HAVE_UNISTD_H 1
 
 /* Define if you have the <wchar.h> header file.  */
-#undef HAVE_WCHAR_H
+#define HAVE_WCHAR_H 1
--- gocr-0.49.orig/src/Makefile
+++ gocr-0.49/src/Makefile
@@ -55,7 +55,7 @@ INSTALL=/usr/bin/install -c
 SHELL=/bin/sh
 
 DEFS=-DHAVE_CONFIG_H
-CPPFLAGS=-Wall -g -fexceptions -DDO_DEBUG=1 
+CPPFLAGS=
 # to see the config.h
 CFLAGS=-g -O2 $(CPPFLAGS) -I../include $(DEFS)
 LDFLAGS=-lm 
@@ -60,7 +60,7 @@ CPPFLAGS=
 CFLAGS=-g -O2 $(CPPFLAGS) -I../include $(DEFS)
 LDFLAGS=-lm 
 LIBS=-lnetpbm 
-DESTDIR=/usr/local
+DESTDIR=/usr
 
 .SUFFIXES: .s .o .c .h
 # do not look for files if help (etc) is given
--- gocr-0.49.orig/man/Makefile
+++ gocr-0.49/man/Makefile
@@ -4,10 +4,10 @@
 
 PROGRAM = gocr
 
-mandir = ${prefix}/man
+mandir = ${prefix}/share/man
 
 INSTALL=/usr/bin/install -c
-DESTDIR=/usr/local
+DESTDIR=/usr
 
 default: all
 
